buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile

```diff
@@ -83,4 +79,13 @@ RUN cd /root \
   && tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2 \
   && rm -f freemarker.tgz
 
+RUN cd /root \
+  && curl -OLJ https://api.adoptopenjdk.net/openjdk8-openj9/releases/s390x_linux/latest/binary \
```

Comment:
Is there a reason to introduce `curl` when the image already has `wget`?

Reply:
no. we can also use `wget`. removed `curl`. changed download link to use `wget`. will use `wget` in #1514 and #1541 as well.